pkg/v1/providers/infrastructure-azure/v1.0.1/yttcc/overlay.yaml

```diff
@@ -110,7 +110,27 @@ spec:
       value: #@ vars[configVariable]
     #@ end
     #@ end
-
+    #@overlay/append
+    - name: TKR_DATA
+      value:
+        v1.23.5+vmware.1:
+          kubernetesSpec:
+            version: v1.23.5+vmware.1
+            imageRepository: #@ bomDataForK8sVersion.kubeadmConfigSpec.imageRepository
+            etcd:
+              imageTag: #@ bomDataForK8sVersion.kubeadmConfigSpec.etcd.local.imageTag
+            coredns:
+              imageTag: #@ bomDataForK8sVersion.kubeadmConfigSpec.dns.imageTag
+          labels:
+            os-name: ubuntu
+            os-type: linux
+            os-arch: amd64
+          osImageRef:
+            sku: spec.image.marketplace.sku
+            publisher: spec.image.marketplace.publisher
+            offer: spec.image.marketplace.offer
+            version: spec.image.marketplace.version
+            thirdPartyImage: spec.image.marketplace.thirdPartyImage
```

Comment:
As we are using this as a dummy data, can we use more accurate dummy values to reduce the confusion for reviewer?